calculator_test.go

```diff
@@ -3,50 +3,43 @@ package calculator_test
 import (
 	"calculator"
 	"math/rand"
+	"strings"
 	"testing"
 	"time"
 )
 
-// GenerateNums it is a helper function
-// that produces a slice of two float numbers.
+// testCese represents input data and expected output
+// from the function under test.
+type testCase struct {
```

Comment:
Fine! But we don't really need to name this struct type, since it tends to be unique to each test. Instead, we can use an anonymous type:

```go
tcs := []struct{
	name string
	...
}{
	{ name: "test case 1"... },
}
```